hash_practice/exercises.py

```diff
@@ -1,19 +1,45 @@
 
+from typing import OrderedDict

```

Comment:
Looks like this `import` statement never got used. In general, just to keep the code clean, you probably want to remove unused `imports` like these.